src/main/kotlin/io/github/mojira/arisa/infrastructure/Cache.kt

```diff
@@ -0,0 +1,37 @@
+package io.github.mojira.arisa.infrastructure
+
+import net.rcarz.jiraclient.Issue
+import java.util.Timer
+import kotlin.concurrent.schedule
+
+class Cache {
+    private val ticketCache = mutableListOf<String>()
+    private val ticketTimer = Timer("RemoveCachedTicket", true)
+    val queryCache = mutableMapOf<String, List<Issue>>()
```

Comment:
Is there any point where you clear this?
This used to be a local variable in execute(), so it used to have a clear queryCache per module